malibal/aon/s1/default.nix

```diff
@@ -0,0 +1,71 @@
+{ pkgs, lib, ... }:
+
+{
+  imports = [
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/cpu/intel
+    ../../../common/gpu/intel
+    ../../../common/gpu/nvidia/disable.nix
+  ];
+
+  hardware = {
+    enableAllFirmware = lib.mkDefault true;
+    enableRedistributableFirmware = lib.mkDefault true;
+
+    graphics = {
+      enable = lib.mkDefault true;
+      enable32Bit = lib.mkDefault true;
+	    extraPackages = with pkgs; [
+	    	(if (lib.versionOlder (lib.versions.majorMinor lib.version) "23.11")
+         then vpl-gpu-rt
+         else intel-vaapi-driver)
+	    ];
```

Comment:
Our intel module enable this.

```suggestion
```

Reply:
I think it's missing the video processing library (`vpl-gpu-rt`),
  - https://github.com/NixOS/nixos-hardware/blob/master/common/gpu/intel/default.nix

I guess the real fix is to add a specific entry for Intel Raptor Lake GPUs and then update the GPUs that are supported by [VPL](https://github.com/intel/vpl-gpu-rt)

Reply:
Yes. If you could do this, this would be the preferred solution.